flow/raster_cache.cc

```diff
@@ -125,7 +125,8 @@ static std::unique_ptr<RasterCacheResult> Rasterize(
     const char* type,
     const std::function<void(SkCanvas*)>& draw_function) {
   TRACE_EVENT0("flutter", "RasterCachePopulate");
-  SkIRect cache_rect = RasterCache::GetDeviceBounds(logical_rect, ctm);
+
+  SkRect cache_rect = RasterCache::GetDeviceBounds(logical_rect, ctm);
```

Comment:
Below we pass width/height to MakeN32Premul. That method takes an integer width/height. I'm not sure why that is compiling, is it performing a silent conversion? Using what method? floor? round? ceil?

Reply:
Ahh shoot, I didn't check and assumed this took an SkScalar. Its probably just truncating?

I think we should probably round out to get the size of the texture, and then when drawing it use the original float offset, right?